lib/stdlib/src/io_lib_pretty.erl

```diff
@@ -853,9 +853,9 @@ printable_bin1(Bin, Start, Len) ->
 
 %% -> all | integer() >=0. Adopted from io_lib.erl.
 printable_latin1_list([_ | _], 0) -> 0;
-printable_latin1_list([C | Cs], N) when C >= $\s, C =< $~ ->
+printable_latin1_list([C | Cs], N) when is_integer(C), C >= $\s, C =< $~ ->
```

Comment:
test case dropped intentionally, due to too much merge conflict to resolve every time.
for ref:
```
github_4801(_Config) ->
         <<"{[81.6]}">> = iolist_to_binary(io_lib:format("~p", [{[81.6]}], [{chars_limit,40}])).
```

Reply:
I can't believe this is still not ported back to latest 23.